src/commands/createNewProject/initProjectForVSCode.ts

```diff
@@ -98,6 +99,10 @@ async function writeVSCodeSettings(projectCreator: ProjectCreatorBase, vscodePat
                 data[`${extensionPrefix}.${preDeployTaskSetting}`] = projectCreator.preDeployTask;
             }
 
+            if (projectCreator.excludedFiles) {
+                data[filesExcludeSetting] = addToFilesExcludeSetting(projectCreator.excludedFiles, vscodePath);
```

Comment:
Don't pass in `vscodePath`. Pass in `data`, which has already been read from the correct settings.json file